modules/iam-activity/main.tf

```diff
@@ -7,6 +7,8 @@ data "streamsec_aws_account" "this" {
 
 locals {
   lambda_source_code_bucket = "${var.lambda_source_code_bucket_prefix}-${data.aws_region.current.name}"
+
+  compatible_runtimes = formatlist(var.lambda_runtime)
```

Comment:
Why is `formatlist` relevant here? it's not formatting anything

Reply:
If you need a list and you have a string you need just `[var.lambda_runtime]` which makes it a single string in a list
`formatlist` is an overkill that is meant for different use case, formatting...

Reply:
⏫